packages/sources/css/src/components/radio-button/src/index.css

```diff
@@ -0,0 +1,93 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+
+body {
+  font-family: var(--vtmn-typo_font-family);
+}
+
+p {
+  margin: 8px 0;
+}
+
+.vtmn-radio-button [type='radio'] {
+  left: 0;
+  opacity: 0.01;
+}
+
+.vtmn-radio-button [type='radio'] + label {
+  position: relative;
+  font-size: var(--vtmn-typo_text-2-font-size);
+  line-height: var(--vtmn-typo_text-2-line-height);
+  cursor: pointer;
+  user-select: none;
+  left: -24px;
```

Comment:
For every pixel property, we need to use "rem()" method. So that the pixel size is automatically generated according to base support (10 or 16).